### ISSUE TITLE
[getmdl.io] ULs without LIs, non-a11y links

### ISSUE THREAD

**Issue by pepelsbey** [google/material-design-lite#1337]
You have a footer on the [getmdl.io](http://www.getmdl.io/) with some problems in it:
```
<footer class="docs-text-styling docs-footer mdl-mini-footer mdl-color--grey-900">
  <ul>
    <a href="https://twitter.com/googledesign" class="mdl-mini-footer--social-btn social-btn social-btn__twitter"></a>
    <a href="https://github.com/google/material-design-lite" class="mdl-mini-footer--social-btn social-btn social-btn__github"></a>
    <a href="https://plus.google.com/+googledesign" class="mdl-mini-footer--social-btn social-btn social-btn__gplus"></a>
  </ul>
</footer>
```
1. Shouldn’t it be `<ul><li><a>` not `<ul><a>` according to spec (zero or more li elements)? If I were you I’d either add `<li>` or remove `<ul>` at all. For educational reasons, at very least.
2. Empty `<a></a>` links are not accessible: it’s not clear where they lead you it you can’t see an icon. It would be good to put some text inside and hide it off-screen.

_Haven’t found a special repository for getmdl.io, so posting it here._

**Comment by surma** [comment]
You were right to open the issue here :)

This is something, #1113 addressed, but only for the templates, so thanks for paying attention. Will fix.
